experiment/manual-trigger/manual-trigger.go

```diff
@@ -0,0 +1,190 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// manual-trigger triggers jenkins jobs based a specified github pull request
+package main
+
+import (
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/url"
+	"strings"
+
+	"k8s.io/test-infra/prow/config"
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/jenkins"
+	"k8s.io/test-infra/prow/kube"
+	"k8s.io/test-infra/prow/pjutil"
+)
+
+type options struct {
+	githubapiurl string
```

Comment:
style nit: camelcase to `githubApiUrl`

Reply:
likewise with the rest of these

Reply:
```
$ golint manual-trigger.go
manual-trigger.go:36:2: struct field githubApiUrl should be githubAPIURL
```

I don't feel strongly either way.  Should I do githubApiURL or githubAPIURL?
Perhaps I should work around it by naming this githubEndpoint